packages/forms/resources/views/components/text-input.blade.php

```diff
@@ -28,11 +29,13 @@
         :suffix-actions="$suffixActions"
         :suffix-icon="$suffixIcon"
         :valid="! $errors->has($statePath)"
+        :is-revealable="$isRevealable"
```

Comment:
For Blade component attributes we follow naming like standard HTML elements, so could you change this to revealable?